transmitter.py

```diff
@@ -8,9 +8,13 @@
 
 
 def encode_byte(x):
+	''' This function encodes the frequency of the audio.
```

Comment:
At all the places, can you please add return types and parameter types as well?
Please refer to the snapshot I have attached in the issue!
Thanks.